docs/conf.py

```diff
@@ -1,133 +1,103 @@
 
-import edx_theme
+# on_rtd is whether we are on readthedocs.org
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
-import django
-from django.utils import six
-
-# Configure Django for autodoc usage
-django.setup()
+if not on_rtd:  # only import and set the theme if we're building docs locally
+    import edx_theme
+    html_theme = 'edx_theme'
+    html_theme_path = [edx_theme.get_html_theme_path()]
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
 # sys.path.insert(0, os.path.abspath('.'))
+REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(REPO_ROOT)
+
+# # Specify settings module (which will be picked up from the sandbox)
+# os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'settings.devstack')
+
+import django
+from django.conf import settings
+settings.configure()
+django.setup()
 
-# -- General configuration ------------------------------------------------
+# -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#
 # needs_sphinx = '1.0'
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-extensions = [
-    'edx_theme',
-    'sphinx.ext.autodoc',
-    'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.ifconfig',
-    'sphinx.ext.autodoc',
-    'sphinxcontrib.napoleon'
-]
-
-# A list of warning types to suppress arbitrary warning messages.
-suppress_warnings = [
-    'image.nonlocal_uri',
-]
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.napoleon']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
-# The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-# source_suffix = ['.rst', '.md']
+# The suffix of source filenames.
 source_suffix = '.rst'
 
 # The encoding of source files.
-#
 # source_encoding = 'utf-8-sig'
 
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
-project = 'enterprise'
-copyright = edx_theme.COPYRIGHT  # pylint: disable=redefined-builtin
-author = edx_theme.AUTHOR
+project = u'Open edX Enterprise Service'
+copyright = u'2016-{year}, edX Inc.'.format(year=datetime.date.today().year)
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = enterprise.__version__
+version = '0.1'
```

Comment:
@mattdrayer Not good. As you can see, everyone just forgets to update it. Please try your best to still keep it.